ui/styles.css

```diff
@@ -0,0 +1,185 @@
+/* Color Pallete 
+    - Navbar #3b8687
+    - Buttons and hovers #F2B134
+    - Text on buttons and navbar: #54b99f
+    - Danger colors : #eb553a
+    - Normal text : # black
+
+    #068587
+*/
+
+/******* 
+Global resets on margin paddins 
+and box sizing
+*******/
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body,
+html {
+    font-family: 'Lato', sans-serif;
+}
+.bg {
+    background: rgb(32, 52, 64);
+    background: linear-gradient(90deg, rgba(32, 52, 64, 1) 0%, rgba(184, 94, 78, 1) 50%, rgba(32, 52, 64, 1) 97%);
```

Comment:
great idea to use color gradient function to create a nice color transition

Reply:
Thanks.
It was tough though getting the right mix of colors.
Luckily i got to use a cool free tool. https://cssgradient.io/ check it out sometime